fenix/app/src/main/java/org/mozilla/fenix/home/topsites/TopSiteItemViewHolder.kt

```diff
@@ -132,11 +89,65 @@ class TopSiteItemViewHolder(
         }
     }
 
+    /**
+     * Binds a top site to the view. This will set the title, url, and image.
+     * setOnClickListener and setOnLongClickListener are also set here.
+     * @param topSite The top site to bind.
```

Comment:
nit: We usually add a newline between the method description and the list of parameters.